src/main/kotlin/io/sip3/captain/ce/pipeline/RtcpHandler.kt

```diff
@@ -155,8 +157,23 @@ class RtcpHandler(context: Context, bulkOperationsEnabled: Boolean) : Handler(co
                 }
             }
 
-            // Move reader index to next RTCP report in packet
-            buffer.readerIndex(offset + reportLength + 4)
+            val nextIndex = offset + reportLength + 4
+            if (nextIndex <= buffer.capacity()) {
+                // Move reader index to next RTCP report in packet
+                buffer.readerIndex(nextIndex)
+            } else {
+                // Stop RTCP packet processing
+                val src = IpUtil.convertToString(packet.srcAddr) + ":${packet.srcPort}"
+                val dst = IpUtil.convertToString(packet.dstAddr) + ":${packet.dstPort}"
+                logger.warn { "Invalid RTCP packet. src $src, dst $dst" }
```

Comment:
I would love to introduce a convention here.
Any message has the following structure:
1. Generic information/description ending with a dot aka `Invalid RTCP packet.`
2. Specific objects/participants, starting from capital letter with `:` aka `Source: $src, Destination: $dst`